citesphere/src/main/webapp/WEB-INF/views/auth/group/editItem.jsp

```diff
@@ -300,3 +343,32 @@ ${author.lastName}<c:if test="${not empty author.firstName}">, ${author.firstNam
     </div>
   </div>
 </div>
+<script>
+$(document).ready(function() {
+	$('#items').on("change", function(e){
+		var itemType = $('#items option:selected').val()
+		$('#displayMessage').replaceWith("<div id='displayMessage' class='alert-info row'>"+
+			"<span class='glyphicon glyphicon-refresh spinning'></span>"+
+				"Loading fields... </div>");
+		$('#displayMessage').show();
+		$.ajax({
+			url : '<c:url value="/auth/items/'+itemType+'/fields" />',
+			type : 'GET',
+			success: function(changedFields){
+				$('form input').parent().closest('tr').hide();
+				var i = 0;
+				for(;i<changedFields.length;i++){
+					$('form input#'+changedFields[i]).parent().closest('tr').show();
+				}
+				$('#displayMessage').hide();
+			},
+			error: function(){
+				$('#displayMessage').replaceWith("<div id='errorDiv' class='alert-danger row'>" +
+					"<i class='glyphicon glyphicon-remove-sign'></i>" +
+						"Error loading the form fields</div>");
+				$('#displayMessage').delay(2000).fadeOut();
+			}
+		});
+	})
+});
```

Comment:
let's make the display message to fit with the rest of the layout, e.g. either grey out the screen or have a blocking modal popup or something.